packages/graphql/src/schemaTypes/objectTypes/gql-Migration.ts

```diff
@@ -227,6 +227,13 @@ export const Migration = objectType({
       },
     })
 
+    t.string('videoEmbedJson', {
+      description: 'stringified JSON from an onlink, containing markup for the migration landing page video embed',
+      resolve: (source, args, ctx) => {
```

Comment:
```suggestion
    t.string('videoEmbedHtml', {
      description: 'Markup for the migration landing page video embed',
      resolve: (source, args, ctx) => {
```

Reply:
Yeah, the case for keeping it as JSON is pretty weak, just the expectation that one day we might add more items to the payload, I'm fine with switching it here.